client/components/data/query-timezones/test/index.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { QueryTimezones } from '../';
+
+describe( 'mounting component', () => {
+	it( 'should mount as null', () => {
```

Comment:
Let's delete this test. The component is simple, and the case we're testing for isn't very interesting